prow/plugins/assign/assign.go

```diff
@@ -138,18 +160,64 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	if len(unassign) > 0 {
-		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
-		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
+		log.Printf("Removing assignees from %s/%s#%d: %v", e.org, e.repo, e.number, unassign)
+		if err := gc.UnassignIssue(e.org, e.repo, e.number, unassign); err != nil {
 			return err
 		}
 	}
 	if len(assign) > 0 {
-		log.Printf("Adding assignees to %s/%s#%d: %v", ae.org, ae.repo, ae.number, assign)
-		if err := gc.AssignIssue(ae.org, ae.repo, ae.number, assign); err != nil {
+		log.Printf("Adding assignees to %s/%s#%d: %v", e.org, e.repo, e.number, assign)
+		if err := gc.AssignIssue(e.org, e.repo, e.number, assign); err != nil {
 			if mu, ok := err.(github.MissingUsers); ok {
 				msg := fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) can be assigned.",
-					strings.Join(mu, ", "), ae.org, ae.org)
-				if e2 := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); e2 != nil {
+					strings.Join(mu, ", "), e.org, e.org)
+				if err2 := gc.CreateComment(e.org, e.repo, e.number, plugins.FormatResponseRaw(e.body, e.url, e.login, msg)); err2 != nil {
+					return fmt.Errorf("comment err: %v", err2)
+				}
+				return nil
+			}
+			return err
+		}
+	}
+	return nil
+}
+
+func handleReview(gc githubClient, log *logrus.Entry, e event) error {
```

Comment:
`handleReview` looks extremely similar to `handleAssign`. Please share code between the two.